app/job_creator.rb

```diff
@@ -6,16 +6,20 @@ def initialize(job_data)
   end
 
   def create_job
-    return if find_job
+    begin
```

Comment:
delete begin-end

```
def create_job
rescue => e
end
```

Reply:
it does not have the begin-end statements in the last version i pushed, u are viewing the outdated one